doc/developer-notes.md

```
@@ -63,7 +63,7 @@ For example, to describe a function use:
  */
 bool function(int arg1, const char *arg2)
 ```
-A complete list of `@xxx` commands can be found at http://www.stack.nl/~dimitri/doxygen/manual/commands.html.
+A complete list of `@xxx` commands can be found at https://www.doxygen.nl/manual/commands.html.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)